test/test_AzureBlobFS.ipynb

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
Please adapt your Notebook to the conventions used in `example/notebooks/datastores` and move it there. Use the same naming convention.

Reply:
It should be named `8_azure_blob filesystem.ipynb`.